RCTOpenTokPublisherView.h

```diff
@@ -15,6 +15,8 @@
 @property (nonatomic, strong) NSString *apiKey;
 @property (nonatomic, strong) NSString *sessionId;
 @property (nonatomic, strong) NSString *token;
+@property (nonatomic, assign) NSInteger cameraResolution;
+@property (nonatomic, assign) NSInteger cameraFrameRate;
```

Comment:
Nevermind, NSInteger is considered a primitive type so using a pointer is not necessary